vpr/src/draw/draw_types.h

```diff
@@ -272,6 +277,12 @@ struct t_draw_state {
 
     std::vector<Breakpoint> list_of_breakpoints;
 
+    //draw state for drawing multiple layers
```

Comment:
Make doxygen compatible: ///@brief
///@brief Stores visibility and transparency drawing controls for each layer [0 .. grid.num_layers-1]